test/js/api/domain/gather-files-spec.js

```diff
@@ -6,55 +7,59 @@ var assert = require('chai').assert,
 
 var util = require('util');
 
-describe('gather-files', function() {
+describe('api/domain/gather-files', function() {
     
     afterEach(function() {
-        stub.restore();    
+        if(stub) stub.restore();   
     });
     
-    var getFiles = function(dir, files, options) {        
-        stub = sinon.stub(glob, "sync", function() {return files.sort()});
-        return gatherFiles(path.normalize(dir), options || {});    
+    var fakePath = '/fake/path';
+    
+    var _getFiles = function(filenames) {
+        return _.map(filenames, function(filename) {return path.join(fakePath, filename);});
+    };
+    
+    var _expected = function(filenames) {
+        return _.map(filenames, function(filename) {
+            return {name: filename, file: path.join(fakePath, filename)};
+        });
+    }
+    
+    var _test = function(files, options) {     
+        stub = sinon.stub(glob, "sync", function() {return files.sort();});
+        return gatherFiles(path.normalize(fakePath), options || {});    
     };
-        
-    it('should return all the files if there is not a matching filename', function() {
-        var expected = [
-            {"name":"a.js","file":"a.js"},
-            {"name":"b.js","file":"b.js"},
-            {"name":"c.js","file":"c.js"}];
-        
-        var actual = getFiles('/fake/dir', ['a.js', 'b.js', 'c.js']);
-        assert.deepEqual(actual, expected);
-    });
 
-    it('should return only the files whose name won\'t match the pattern', function() {
-        var expected = [{"name":"a.js","file":"a.js"}, {"name":"c.js","file":"c.js"}];        
-        var actual = getFiles('/fake/dir', ['a.js', 'c.js', 'response_b.js']);
-        assert.deepEqual(actual, expected);
+    it('should recognize responses as files starting with \'response_\' by default', function() {
+        assert.deepEqual(
+            _test(_getFiles(['request_a', 'response_a', 'request_b', 'res_b'])), 
+            _expected(['request_a', 'request_b', 'res_b']));
```

Comment:
I mean gatherFiles doesn't return a list of file names but a list of objects with the format I said before. If you use the correct format it will help those reading the code to know what to expect from gatherFiles